README.md

```diff
@@ -4,6 +4,9 @@ pytcg
 Status: Very early, but working!  So far it's able to translate from 32-bit x86
 to TCG ops and print out some op info. Pythonification is next!
 
+Now it's accessible from the following:
```

Comment:
Does the `pip install` do everything? If yes then I prefer to restructure the README.md to reflect that the user can pick either the automatic method or manual install.

Reply:
yes, right now it do all the thing at once.